src/touch-unit.sources

```diff
@@ -1,273 +1,273 @@
 NUNITLITE_SOURCES = \
-	$(NUNITLITE_PATH)/NUnitLite-1.0.0/src/framework/AssemblyInfo.cs \
```

Comment:
You could have just set `NUNITLITE_PATH=$(MONO_IOS_SDK_DESTDIR)/ios-sources/external/nunit-lite` at the top of this file and this would be a one-line change :smile: Still, it's fine as it is.